src/screen.rs

```diff
@@ -142,7 +142,7 @@ impl Screen {
             let pid = new_pids.next().unwrap(); // if this crashes it means we got less pids than there are panes in this layout
             let mut new_terminal = TerminalPane::new(
                 *pid,
-                self.full_screen_ws.clone(),
+                self.full_screen_ws,
```

Comment:
@imsnif clippy really does not like the `.clone()` here, is it necessary?

Reply:
I think this is `Copy`, so no need for sure. It's probably a leftover from a refactor.

Reply:
😎